components/dot_menu/dot_menu.jsx

```diff
@@ -288,6 +291,37 @@ export default class DotMenu extends React.PureComponent {
                 );
             });
 
+        const pluginIntegrationsItems = this.props.pluginIntegrations.filter((item) => {
+            return item.location_type === PluginLocation.PLUGIN_LOCATION_POST_ACTION;
```

Comment:
Looks like the parent already does this filtering with
`getPluginIntegrations(state, PluginLocation.PLUGIN_LOCATION_POST_ACTION);`